main/main.cpp

```diff
@@ -1134,6 +1139,19 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 				OS::get_singleton()->print("Missing fixed-fps argument, aborting.\n");
 				goto error;
 			}
+		} else if (I->get() == "--write-movie") {
+			if (I->next()) {
+				write_movie_path = I->next()->get();
+				N = I->next()->next();
+				if (fixed_fps == -1) {
+					fixed_fps = 60;
+				}
+			} else {
+				OS::get_singleton()->print("Missing write-movie argument, aborting.\n");
+				goto error;
+			}
+		} else if (I->get() == "--disable-vsync") {
+			disable_vsync = true;
```

Comment:
See also https://github.com/godotengine/godot-proposals/issues/1923. I think the way this PR handles V-Sync disabling is fine, just something to keep in mind for the future. I'll definitely appreciate having this CLI argument available for games that don't expose a way to turn off V-Sync :slightly_smiling_face: